### ISSUE TITLE
[Feature Request] Hide categories in the AppGraber automatically/manually

### ISSUE THREAD

**Issue by PeterStrick** [cairoshell/cairoshell#395]
**Is your feature request related to a problem? Please describe.**
Sometimes if you have a bunch of applications added to the Programs menu in Cairo and you want to add another program, you get a messy list in the App Graber that lists all your Programs in every section/category. 

**Describe the solution you'd like**
Hiding the Programs in the AppGraber by collapsing the sections/categories. Then Uncollapse the section/category when pressed on it.

**Describe alternatives you've considered**
Press the Up/Down arrows to move the section/category to the top for easier access.

**Additional context**
A picturehow I imagine that to work is added here: (red text is me explaining)
![Anmerkung 2020-06-14 180618](https://user-images.githubusercontent.com/60312421/84598388-9ce15180-ae6a-11ea-9c3e-96152439be22.png)

**Comment by dremin** [comment]
This is not the first report of confusion regarding the "Hide" button that we have seen. If we used expand/collapse arrows on each category instead, would that have made more sense to you in the moment, and provide the flexibility you're looking for?

**Comment by PeterStrick** [comment]
> This is not the first report of confusion regarding the "Hide" button that we have seen. If we used expand/collapse arrows on each category instead, would that have made more sense to you in the moment, and provide the flexibility you're looking for?

Yeah something like that makes more sense. Expand/collpase buttons for each category individually are better in my opinion.

**Comment by jerry-monitor** [comment]
I was very confused about the Hide button. Is that the Collapse button as it is called in English?

Maybe this list would work better if it was in a tree structure where you can collapse and expand categories, click F2 on the title to edit the name, drag and drop a category to change the order, and maybe even drag and drop the order of the items within a category instead of sorting alphabetically.
